.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -19,111 +19,65 @@ body:
         - label: I searched the issue tracker but was unable to find my issue.
           required: true
         - label: |
-            I read
-
-              - the [extended documentation in general](https://docker-mailserver.github.io/docker-mailserver/latest/) but found nothing to resolve the issue;
-              - the [documentation on debugging](https://docker-mailserver.github.io/docker-mailserver/latest/config/debugging/), tried the proposed steps to debug the problem, but was still unable to resolve the issue;
-              - this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree;
-              - the [documentation about filing a bug report](https://docker-mailserver.github.io/docker-mailserver/latest/contributing/issues-and-pull-requests/#filing-a-bug-report).
+            I have read:  
```

Comment:
I intentionally added two spaces at the end of this line, thinking it might help with the initial formatting.

When filing the issue, it's not properly markdown formatted into multi-line list, only after submission. Seems to be due to a restriction on `label` perhaps? Whereas if it was in `description` it'd be rendered as a list.

Reply:
> I intentionally added two spaces at the end of this line, thinking it might help with the initial formatting.

I noticed this behavior as well. Does your fix with the spaces improve the situation? The preview is still a bit gobbled I think..

Reply:
> Does your fix with the spaces improve the situation?

:man_shrugging: I haven't tried it out on a test repo sorry :sweat_smile: 

A future revision can move this into `description`, with the first two checkbox items being added to debug docs.

Reply:
Merged, it did not help :cry: 

Form looks better overall though I think :sunglasses: